tap_frontapp/__init__.py

```diff
@@ -23,14 +23,27 @@
 #  call to the api but with the odd frontapp structure, we won't do that
 #  here we never use atx in here since the schema is from file but we
 #  would use it if we pulled schema from the API def discover(atx):
+
+
 def discover():
     catalog = Catalog([])
+    
+    # Build initial catalog from schema files
     for tap_stream_id in schemas.STATIC_SCHEMA_STREAM_IDS:
-        #print("tap stream id=",tap_stream_id)
+        # LOGGER.info("tap stream id=%s", tap_stream_id)
```

Comment:
Uncomment the statement.
```suggestion
        # LOGGER.info("tap stream id=%s", tap_stream_id)
```